### PR TITLE
Python support added for 3.9, 3.10, 3.11, 3.12

### PR DESCRIPTION
**Pull request by MehrazRumman**

### CODE REVIEW

**Comment by MehrazRumman** [comment]
@auvipy  can you please review this pr !

**Comment by MehrazRumman** [comment]
@auvipy  I think its ready for review now

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9
```
Comment (auvipy) [inline .travis.yml:12]: You can undo changes in this file

**Review by auvipy — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (MehrazRumman) [reply .travis.yml]: I undo this changes as per you requested !

**Comment by MehrazRumman** [comment]
@auvipy

**Review by auvipy — APPROVED** [review]

**Pull request merged**